tfjs-backend-webgpu/src/kernels/reduce_webgpu.ts

```diff
@@ -77,66 +80,68 @@ export class ReduceProgram implements WebGPUProgram {
         `setOutput(flatOutputIndex, bestValue);`;
 
     const sharedMemorySnippet = `
-        shared float xBestValues[WorkGroupSize];
-      `;
+         shared float xBestValues[WorkGroupSize];
+       `;
     const sharedMemoryReduceSnippet = `
-      xBestValues[gl_LocalInvocationID.x] = bestValue;
-      ${this.reduceType === 'sum' || this.reduceType === 'mean' ||
-          this.reduceType === 'prod' ? `bestValue=${initValue};` : ' '}
-      int currentSize = WorkGroupSize;
-      while (currentSize > 1) {
-        barrier();
-        for (int w = 0; w < ${this.reductionFactor}; ++w) {
-          int i = int(gl_LocalInvocationID.x) * ${this.reductionFactor} + w;
-          if (i < currentSize) {
-            float candidate = xBestValues[i];
-            ${reduceOp}
-          }
-        }
-        barrier();
-        xBestValues[gl_LocalInvocationID.x] = bestValue;
-        currentSize = DIV_CEIL(currentSize, ${this.reductionFactor});
-        ${this.reduceType === 'sum' || this.reduceType === 'mean' ||
-            this.reduceType === 'prod' ?
-            `if(currentSize > 1) bestValue=${initValue};` : ''}
-      }
-      if (gl_LocalInvocationID.x == 0) {
-        ${outputSnippet}
-      }
-    `;
+       xBestValues[gl_LocalInvocationID.x] = bestValue;
+       ${
+        this.reduceType === 'sum' || this.reduceType === 'mean' ||
+                this.reduceType === 'prod' ?
+            `bestValue=${initValue};` :
+            ' '}
+       int currentSize = WorkGroupSize;
+       while (currentSize > 1) {
+         barrier();
+         for (int w = 0; w < ${this.reductionFactor}; ++w) {
+           int i = int(gl_LocalInvocationID.x) * ${this.reductionFactor} + w;
+           if (i < currentSize) {
+             float candidate = xBestValues[i];
+             ${reduceOp}
+           }
+         }
+         barrier();
+         xBestValues[gl_LocalInvocationID.x] = bestValue;
+         currentSize = DIV_CEIL(currentSize, ${this.reductionFactor});
+         ${
+        this.reduceType === 'sum' || this.reduceType === 'mean' ||
+                this.reduceType === 'prod' ?
+            `if(currentSize > 1) bestValue=${initValue};` :
+            ''}
+       }
+       if (gl_LocalInvocationID.x == 0) {
+         ${outputSnippet}
+       }
+     `;
```

Comment:
Are these codes only format issue? If so, could we upload another CL only fix format issue, if so, it will be more readable for this CL.

Reply:
This is done by the Visual studio code automatically.